go/vt/vtgate/engine/delete.go

```diff
@@ -131,6 +131,9 @@ func (del *Delete) description() PrimitiveDescription {
 	}
 
 	addFieldsIfNotEmpty(del.DML, other)
+	if del.FetchLastInsertID {
+		other["FetchLastInsertID"] = del.FetchLastInsertID
+	}
 
```

Comment:
AFAIK, we do not need to check the boolean, if it is false, it is ignored from the json output

Reply:
you need to mark the json with `omitempty` for that to be true. I just tried, and it's definitely not ignored

Reply:
One of these days, I will make it so.